csrankings.csv

```diff
@@ -15787,3 +15787,56 @@ Zygmunt J. Haas,University of Texas at Dallas,http://www.utdallas.edu/~zjh130030
 Ümit V. Çatalyürek,Georgia Institute of Technology,https://www.cc.gatech.edu/~umit,OLDMURQAAAAJ
 Mayank Singh 0001,IIT Gandhinagar,http://www.iitgn.ac.in/faculty/comp/mayank.htm,FOjZrQcAAAAJ
 Nipun Batra 0001,IIT Gandhinagar,https://www.iitgn.ac.in/faculty/comp/nipun.htm,rFGzHlIAAAAJ
+Robert I. Davis,University of York,https://www-users.cs.york.ac.uk/~robdavis/,xjVBcr4AAAAJ
+Iain Bate,University of York,https://www-users.cs.york.ac.uk/~ijb/,dE_ooMIAAAAJ
+Leandro Soares Indrusiak,University of York,https://www-users.cs.york.ac.uk/~lsi/,f1PUDsEAAAAJ
+Wanli Chang,University of York,https://www-users.cs.york.ac.uk/~wanli/,OYcT_B4AAAAJ
+Andy J. Wellings,University of York,https://www-users.cs.york.ac.uk/~andy/,yPa4QrkAAAAJ
+Neil C. Audsley,University of York,https://www-users.cs.york.ac.uk/~neil/,D65ZWSgAAAAJ
+Ian Gray,University of York,https://www-users.cs.york.ac.uk/~iang/,HloK4CAAAAAJ

```

Comment:
Web page should be https://iangray.me.uk/

Reply:
No, this is the correct address. I may move my hosting at some point but whilst I am a member of the department this is my web address.